plugins/pipelines/app/views/samson_pipelines/_stage_show.html.erb

```diff
@@ -19,7 +19,10 @@
     <div>
       After this stage finishes deploying it will deploy to:
       <%= unordered_list next_stages do |stage| %>
-        <%= link_to stage.name, [@project, stage] %>
+        <%= link_to stage.name, [@project, stage] %>. 
+        <% if last_deploy = stage.last_deploy %> 
+          Last Deploy: <%= render_time(last_deploy.start_time) %> <%= status_badge last_deploy.status %>
```

Comment:
Maybe drop the parens on `render_time` to be consistent?

Reply:
👍